plugins/vli/fu-vli-usbhub-msp430-device.c

```diff
@@ -149,8 +149,7 @@ fu_vli_usbhub_msp430_device_detach (FuDevice *device, GError **error)
 
 	/* avoid power instability by waiting T1 */
 	fu_device_set_status (device, FWUPD_STATUS_DEVICE_RESTART);
-	fu_device_set_progress (device, 0);
-	g_usleep (G_USEC_PER_SEC);
+	fu_device_msleep_with_progress (device, 4000);
```

Comment:
Didn't you bump up the sleep to longer here

Reply:
Eek, fixed.